src/zenml/steps/step_context.py

```diff
@@ -184,3 +184,22 @@ def get_output_artifact_uri(self, output_name: Optional[str] = None) -> str:
             Artifact URI for the given output.
         """
         return self._get_output(output_name).artifact_uri
+
+    @staticmethod
+    def _get_context_run_time() -> PipelineRunResponseModel:
+        """
+        Returns: PipelineRunResponseModel
+        """
+
+        pipeline_run_id = Environment().step_environment.pipeline_run_id
+        pipeline_run = Client().get_pipeline_run(pipeline_run_id)
+        return pipeline_run
+
+    def get_context_run_time(self) -> datetime.datetime:
```

Comment:
I think a more fitting name would be something like `get_run_start_time`? Run time (at least to me) seems more like the duration of the run, which this method does not return